src/frontend/src/optimizer/plan_node/stream_table_scan.rs

```diff
@@ -205,6 +206,13 @@ impl StreamTableScan {
             column_ids: upstream_column_ids.clone(),
         };
 
+        let properties = self.ctx().with_options().internal_table_subset(); // TODO: Is this even needed? Why is it needed for simple_agg?
```

Comment:
Why is `with_options` needed for agg internal state?
Is it needed for backfill internal state too?

Reply:
IIRC, TTL options are passed in this way to clean the state.

Reply:
What is meant by "ttl options" here?